packages/barcode-scanning/src/definitions.ts

```diff
@@ -382,6 +390,24 @@ export interface GetMaxZoomRatioResult {
   zoomRatio: number;
 }
 
+/**
+ * @since 8.0.0
+ */
+export interface SetFocusPointOptions {
+  /**
+   * X coordinate (0.0 = left, 1.0 = right)
+   *
+   * @since 8.0.0
+   */
+  x: number;
```

Comment:
Let's make this property optional and the default value should be 0.5. What do you think? Then we can remove a few checks and exceptions in the native layer.

Reply:
that's a great idea - will do